authentication/jwt_session.py

```
@@ -0,0 +1,220 @@
+        cached = cache.get(grace_key)
+        if cached:
+            return json.loads(cached)
+
+        # Check user is still active
+        user_id = refresh.get(api_settings.USER_ID_CLAIM)
+        if user_id:
+            user = User.objects.get(pk=user_id)
+            if not api_settings.USER_AUTHENTICATION_RULE(user):
+                raise AuthenticationFailed(
+                    "No active account found for the given token."
+                )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)